src/SelectList/SelectList.tsx

```diff
@@ -12,11 +12,8 @@ export interface SelectListItem {
 }
 
 export interface SelectListProps {
-  value?: string
```

Comment:
I'm not too familiar with this code, but curious for why these props were removed?